### PR TITLE
[TASK] Refine labels for 'General Web Application' settings

### PR DESCRIPTION
**Pull request by leechwin**
[DESC.]
Related with https://github.com/webida/webida-client/pull/355.
Combining the labels and symbols.

Signed-off-by: ChangHyun Lee <leechwin.lee@samsung.com>

### CODE REVIEW

File: apps/ide/src/plugins/webida.ide.project-management.run/layout/default-run-configuration.html [hunk 1]
```diff
@@ -22,14 +22,14 @@
                 </div>
             </div>
             <div class='rcw-table-row'>
-                <div class='rcw-content-table-label rcw-table-col'>Query String</div>
-                <div class='rcw-content-table-value rcw-table-col'>?
```
Comment (kyungmi) [inline apps/ide/src/plugins/webida.ide.project-management.run/layout/default-run-configuration.html]: I think the previous version of label is more clear to understand the intent.

Comment (leechwin) [inline apps/ide/src/plugins/webida.ide.project-management.run/layout/default-run-configuration.html]: OK. I agree with you.

Comment (kyungmi) [inline apps/ide/src/plugins/webida.ide.project-management.run/layout/default-run-configuration.html]: This issue will be continued #375

**Comment by kyungmi** [comment]
@leechwin This issue of ordering will be followed up on #376

**Pull request merged**